koku/api/report/ocp_aws/ocp_aws_query_handler.py

```diff
@@ -40,18 +40,27 @@ def __init__(self, query_parameters, url_data,
             kwargs    (Dict): A dictionary for internal query alteration based on path
         """
         provider = 'OCP_AWS'
+        self._initialize_kwargs(kwargs)
         if kwargs.get('access'):
-            query_parameters = update_query_parameters_for_openshift(query_parameters, kwargs.get('access'))
-        super().__init__(query_parameters, url_data, tenant,
-                         provider=provider, **kwargs)
+            query_parameters = update_query_parameters_for_openshift(query_parameters,
+                                                                     kwargs.get('access'))
+
+        self._mapper = OCPAWSProviderMap(provider=provider,
+                                         report_type=kwargs.get('report_type'))
+        self.group_by_options = self._mapper.provider_map.get('group_by_options')
+        self.query_parameters = query_parameters
+        self.url_data = url_data
+        self._limit = self.get_query_param_data('filter', 'limit')
 
         # Update which field is used to calculate cost by group by param.
         group_by = self._get_group_by()
         if (group_by and group_by[0] == 'project') or \
                 'project' in self.query_parameters.get('filter', {}).keys():
             self._report_type = self._report_type + '_by_project'
```

Comment:
Is `self._report_type` set anywhere before this? In the OCP file it looks like we reference `kwargs.get('report_type')` instead.

Reply:
Good catch, that line is not quite right.

Fixed.